packages/hooks/src/useCountDown/index.ts

```diff
@@ -19,7 +20,8 @@ const calcLeft = (t?: TDate) => {
   if (!t) {
     return 0;
   }
-  const left = new Date(t).getTime() - new Date().getTime();
+  // https://stackoverflow.com/questions/4310953/invalid-date-in-safari
+  const left = new Date(dayjs(t).toDate()).getTime() - new Date().getTime();
```

Comment:
是不是直接用 dayjs 的 `.valueOf()` 就可以了

```suggestion
  const left = dayjs(t).valueOf() - dayjs().valueOf();
```